rfcs/text/0000-rfc-resource-tagging.md

```diff
@@ -0,0 +1,200 @@
+`project`   | The value for the project key shall be `dcp`
+`env`       | The value for the env key shall be the stage that the resource applies to, this can be `{dev,staging,integration,prod}`
+`owner`     | The email of a team or individual responsible for operating the asset, for example: dss-team@data.humancellatlas.org
+`service`   | The name of the HCA DCP service that manages the resource, for example: DSS, IngestService
+`Name`      | A human-readable name for the asset. This can be a custom name or a combination of other tags, such as `dss-notify-dev` (service-subcomponent-env). The name should identify the service subcomponent that the asset belongs to, if any. (The value of this tag shows up as the asset name in the AWS Console UI.)
+`managedBy` | The name of a deployment management tool managing this asset, for example: `terraform`, `cloudformation`, `chalice`
+
+
+#### Termination policy
+Assets that are not tagged in accordance with the above are subject to termination.
+
+* After this RFC takes effect, a 90 day grace period will apply, in which all DCP services are expected to implement tagging.
+* After the grace period, an automated "reaper" process will periodically flag untagged assets and announce this in the
+  `#dcp-ops-alerts` channel on the HCA Slack.
+* Any resources mentioned in such an announcement that cost over $100/day will be subject to immediate termination.
+* Any other resources mentioned in such an announcement will be terminated after a 7 day grace period elapses.
+
+#### Role session name (AWS only)
+To facilitate auditing of operator actions, all operators **MUST** configure a `role_session_name` parameter in their AWS CLI
+configuration.
+
+* See the
+  [HCA DCP Ops Wiki](https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/Setting%20up%20the%20AWS%20Console%20and%20AWS%20CLI%20to%20work%20with%20HCA%20DCP)
+  for more information regarding `role_session_name`.
+* The [AWS-managed tag `aws:createdBy`](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/aws-tags.html)
+  incorporates the resulting principal name, and can be used to track resources in AWS auditing tools, including the
+  Cost Explorer.
+* After this RFC takes effect, a 90 day grace period will apply, in which all DCP developers and operators are expected to
+  configure AWS role session names for their accounts.
+* After the grace period, access may be denied when executing AssumeRole without an identifiable session name.
+
+#### Example: Tagging with Terraform
+The following is an example of a default set of tags that can be applied to an AWS Resource in Terraform.
+Using this local.common_tags allows these tags to be defined in one section for the variables, and be used multiple times.
+If a resource requires an additional independent tag, this can also be achieved.
+
+```
+// variables.tf
+locals {
+  common_tags = "${map(
+    "managedBy" , "terraform",
+    "Name"      , "${var.project}-${var.env}-${var.service}",
```

Comment:
@kislyuk this label is not gcp compatable, no uppercase characters are supported, `name` is compatible